commander/src/bootstrapping/commands/generator/export.ts

```diff
@@ -69,6 +69,7 @@ export abstract class ExportCommand extends BaseIPCClientCommand {
 	static description = 'Export to <FILE>.';
 
 	static examples = [
+		'generator export',
```

Comment:
```suggestion
		'generator:export',
```

Although `generator export` will also work fine, in all other examples, commands use colon to separate topic from command.

Reply:
With the oclif update, the space is also supported in all commands.